image/base/nginx-1.26.1.tar.gz.asc

```diff
@@ -0,0 +1,17 @@
+-----BEGIN PGP SIGNATURE-----
```

Comment:
Downloaded from https://nginx.org/en/download.html

Reply:
It is safe to retrieve this signature from the web if you would find that to be more convenient.  (Consider that when this is next bumped, the signature will change.)

Ensuring trust on the key is of sole importance.
With the key vendored, we can use that to verify everything else.
(As a bonus, the key -- or keys -- should not change between releases.)